packages/react-text/src/Text.stories.tsx

```diff
@@ -1,24 +1,27 @@
 import { makeStyles } from '@fluentui/react-make-styles';
 import * as React from 'react';
+import { Meta } from '@storybook/react';
```

Comment:
(Comment on the whole file) Per the [story authoring RFC](https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/authoring-stories.md#8-location-and-naming-convention), this stories file and probably also the markdown files should probably be under `src/components/Text`.

Reply:
Thanks for pointing that out. Although, from the updates of the documentation so far, it is not clearly specified where the stories should be. I see that some `.stories.tsx` files have been placed under `src/stories/` or under `src/componentName`. Maybe @PeterDraex could clear out where the `.stories.tsx` should be placed?

Reply:
Ah the `src/stories`  culprit is me. I would say you can use some personal judgement there.... Menu has about about 20+ files related just to stories, and it seeme inapproriate to add them all to the root `/src`  directory. It' s smth that I've also brought to the attention of @PeterDraex

Reply:
Agree, thanks for the input ^_^ IMO the Text it is better to have it under the `src/` where also the `TextWrappers.stories.mdx` is placed, so everything related to stories to be in one place. I'll wait also for @PeterDraex's opinion on this.

Reply:
The RFC is a bit out out of date. I apologize for that. I will propose changes to it. 

See this comment:
https://github.com/microsoft/fluentui/issues/19389#issuecomment-906590912

It's okay to place all .stories.tsx files into a stories/ folder. Location of this folder is up to you.

Reply:
The RFC is a bit outdated, sorry for that. See this comment:
https://github.com/microsoft/fluentui/issues/19389#issuecomment-906590912

Reply:
Np, I will go with the option of the `stores/` folder as it makes more sense here. Thanks